File: android/OhDormitory/app/src/main/java/kr/hs/emirim/uuuuri/ohdormitory/Model/BasicNotice.java
package kr.hs.emirim.uuuuri.ohdormitory.Model;

public class BasicNotice extends Notice {

    private String content;

    public BasicNotice(int notice_id, String title, String w_time, int type, String content) {
        super(notice_id, title, w_time, type);
        this.content = content;
    }

    public String getContent() {
        return content;
    }
}

File: ios/OhDormitory/OhDormitory/BasicNotice.swift
//
//  BasicNotice.swift
//  OhDormitory
//
//  Created by 장명수 on 2018. 6. 11..
//  Copyright © 2018년 doori. All rights reserved.
//

import Foundation

class BasicNotice : Notice{
    var content : String
    init(notice_id:Int, title:String, type:Int, w_time:String, d_time:String, content:String){
        self.content = content
        super.init(notice_id: notice_id, title: title, type: type, w_time: w_time, d_time: d_time)
    }
}

